### PR TITLE
Rearranged Order of Instances 

### PR DESCRIPTION
**Pull request by EdwardLangdon**
Fixes ordering issue made by merging #168

### CODE REVIEW

**Comment by ManeraKai** [comment]
You removed `https://piped.mha.fi`

File: src/assets/javascripts/helpers/youtube/youtube.js [hunk 1]
```diff
@@ -91,14 +91,13 @@ let redirects = {
       "https://piped.kavin.rocks",
       "https://piped.silkky.cloud",
       "https://piped.tokhmi.xyz",
+      "https://piped.moomoo.me",
       "https://il.ax",
       "https://piped.syncpundit.com",
       "https://piped.mha.fi",
```
Comment (EdwardLangdon) [inline src/assets/javascripts/helpers/youtube/youtube.js:97]: @ManeraKai I didn't remove it is ordered as in the Official list. See this line.

Comment (EdwardLangdon) [inline src/assets/javascripts/helpers/youtube/youtube.js:97]: @ManeraKai I didn't remove it is ordered as in the Official list. See this line

Comment (EdwardLangdon) [inline src/assets/javascripts/helpers/youtube/youtube.js:97]: @ManeraKai I didn't remove it is ordered as in the Official list. See this line. It's still there.

Comment (EdwardLangdon) [inline src/assets/javascripts/helpers/youtube/youtube.js:97]: @ManeraKai I didn't remove it is ordered as in the Official list. See this line. It's still there.

**Comment by EdwardLangdon** [comment]
@ManeraKai Why not push it now ?

**Comment by ManeraKai** [comment]
I commented on your comment. See it

**Pull request merged**